griidc/pelagos
# refactor/nfsv4 acl compat

To test:

The NetApp is already connected to griidc-dev, so this branch needs to be merged into anything else you're working on if you need files to work.

1. create/approve new dif, then submit a dataset with a small file to test.
2. Approve the submission, then download it from dataland.
3. Repeat for http pull and sftp uploaded file.

Check permissions on:
/san/data/store/FOLDER
/san/data/store/FOLDER/DATAFILE
/san/data/download/FOLDER
/san/data/download/FOLDER/DATAFILE

In order to see the NFS ACL permissions, use the nfs4_getfacl command.
pertinent info: 48 = UID of 'apache' user, 1008 = GID of 'griidc' ldap group.
r=read, x=execute (not traversal as with UNIX), t = traversal
